driver/src/main/java/org/neo4j/driver/internal/metrics/ListenerEvent.java

```diff
@@ -18,10 +18,18 @@
  */
 package org.neo4j.driver.internal.metrics;
 
-
 public interface ListenerEvent
 {
     void start();
+
     long elapsed();
+
+    interface PoolListenerEvent extends ListenerEvent
```

Comment:
Why are these empty interfaces needed? Can't `ListenerEvent` be used everywhere?

Reply:
For 
`MetricsListener#void afterAcquiredOrCreated( BoltServerAddress serverAddress, PoolListenerEvent acquireEvent );`
and
`void afterAcquiredOrCreated( BoltServerAddress serverAddress, ConnectionListenerEvent acquireEvent );`

It is also more clear to use.

Reply:
`ListenerEvent.PoolListenerEvent` and `ListenerEvent.ConnectionListenerEvent` are the same as `ListenerEvent`. I do not really see the reason for having them right now. They only force us to type more :)